request_profiler/models.py

```diff
@@ -153,6 +153,10 @@ class ProfilingRecord(models.Model):
     request_uri = models.URLField(
         verbose_name="Request path"
     )
+    query_string = models.TextField(
+        null=True,
+        verbose_name="Query parameters string"
```

Comment:
A two word name would be more concise and sound better.

```suggestion
        verbose_name="Query string"
```